pkg/engine/engine.go

```diff
@@ -115,6 +115,8 @@ func (e *Engine) Render(chrt *chart.Chart, values chartutil.Values) (map[string]
 type renderable struct {
 	// tpl is the current template.
 	tpl string
+	// path is the path of chart
+	path string
```

Comment:
https://github.com/kubernetes/helm/blob/15576abd9aaf7f285a7701905dbafbb256161c87/pkg/engine/engine.go#L127

I can merge this PR anyway to unblock any ongoing work, but would like to get ur thoughts first.